src/models/project.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+class Project {
+    constructor(name, startDate, endDate, estimatedHours) {
+        this._name = name;
+        this._startDate = startDate;
+        this._endDate = endDate;
+        this._estimatedHours = estimatedHours;
+        this._developers = [];
+    }
+
+    addDeveloper(developer) {
+        if (!this._developers.includes(developer)) {
+            this._developers.push(developer);
+            developer.addProject(this);
+        }
+    }
+
+    activeDevs(weekNumber) {
+        let result = [];
+
+        this._developers.forEach((dev) => {
+            if (dev.vacationDays.length > 0) {
+                dev.vacationDays.forEach((date) => {
+                    if (date.getWeek() != weekNumber) {
+                        result.push(dev);
```

Comment:
Kan dette ikke gøre så samme dev kan være flere gange i listen af aktive devs? Hvis nu dev'en har 2 ugers ferie, som ikke er den valgte uge?

Reply:
Jo, du har ret. Laver en break efter. Hvilket burde fixe problemet.

Reply:
Øh, nej ikke når du bruger forEach metoden. Du kan i stedet for dit forEach() bruge [some()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) for at tjekke om en given dev har ferier der. så noget i den her stil ville måske virke:
```JS
if(dev.vacationDays.som(date => date.getWeek() != weekNumber)) {
    result.push(dev);
}
```